react/features/toolbox/functions.native.js

```diff
@@ -13,6 +14,7 @@ export function isToolboxVisible(stateful: Object | Function) {
     const state = toState(stateful);
     const { alwaysVisible, enabled, visible } = state['features/toolbox'];
     const { length: participantCount } = state['features/base/participants'];
+    const flag = getFeatureFlag(state, TOOLBOX_ALWAYS_VISIBLE);
```

Comment:
can you add the `false` default? 

```suggestion
    const flag = getFeatureFlag(state, TOOLBOX_ALWAYS_VISIBLE, false);
```

Reply:
Done